src/datarobotx/idp/retraining_policies.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2024 DataRobot, Inc. and its affiliates.
+#
+# All rights reserved.
+#
+# DataRobot, Inc.
+#
+# This is proprietary source code of DataRobot, Inc. and its
+# affiliates.
+#
+# Released under the terms of DataRobot Tool and Utility Agreement.
+# https://www.datarobot.com/wp-content/uploads/2021/07/DataRobot-Tool-and-Utility-Agreement.pdf
+
+import json
+from typing import Any, Dict, Union
+
+import datarobot as dr
+
+
+def _check_response(response: Any) -> None:
+    if response.status_code < 200 or response.status_code > 399:
```

Comment:
Any reason this is not a specific number?

Reply:
Yes, [https://sagrawal003.medium.com/http-response-status-codes-successful-200-299-grow-together-by-sharing-knowledge-57a35586c65f#:~:text=Successful%20responses%20(status%20codes%20200,%2C%20resource%20creation%2C%20and%20more.](article on http codes). I think it should actually be >299 now that I think about it.